public/locales/ru/translations.json

```diff
@@ -0,0 +1,183 @@
+{
+  "language name": "Русский",
+
+  "home": "Главная",
+  "recent": "Недавняя Активность",
+  "no followers": "Ни одного подписчика",
+  "latest followers": "Последние Подписавшиеся",
+  "connect wallet to view": "Подключите кошелек, чтобы увидеть",
+  "phrase": "Все о тех, кого вы знаете. 😜",
```

Comment:
The phrase "It's about who you know." in Russian sounds silly. Made some minor changes, now it sounds like "All About People You Know".